Assets/FireThrower.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+
+public class FireThrower : MonoBehaviour
+{
+    [SerializeField] private ParticleSystem fireParticles;
+    [SerializeField] private BoxCollider fireCollider;
+    [SerializeField] private bool isActive = true;
+
+    private void Start()
+    {
+        if (isActive)
```

Comment:
Use fonction ToggleFire() directly + script is not in Scripts/ folder

Reply:
Can't use it, it's doing the opposite of the toggle fire here !
And I'll move the file